docs/usage/getting-started/running.md

```diff
@@ -1,70 +1,97 @@
 If you need Renovate to support any non-npm lock files like Bundler then you'll need to make sure all required third-party tools are pre-installed in the same environment alongside Renovate before you run it.
 
 The `renovate` npm package is compatible with all of Renovate's supported platforms.
 
-Renovate requires Node.js `>=14.15.0` and Git `>=2.33.0`.
+Renovate requires Node.js `>=18.12.0` and Git `>=2.33.0`.
+
+#### Docker images
+
+Renovate is also distributed as Docker images on Docker Hub (`renovate/renovate`) and GitHub container registry (`ghcr.io/renovatebot/renovate`).
+These Docker images work on all the hosting platforms that Renovate supports.
+
+You must run the Docker images in a `linux/amd64` container.
+We're working on `linux/arm64` container support.
+You can't run the Docker images on a Windows or macOS container.
+
+In general, you can run Renovate on Windows with global mode.
+
+There are two flavors:
 
-#### Docker image
+- The `-slim` image
+- The default "full" image, this is the image you'll get if you use the `latest` tag
 
-The `renovate` npm package is also distributed via pre-built Node.js images on Docker Hub (`renovate/renovate`).
+##### The slim image
 
-The `slim` image contains only Node.js so works if either:
+The `-slim` image only comes with the Node.js environment.
+By default, the image installs the required tools when needed.
+We recommend the `-slim` image for most users.
 
-- You do not require any additional package managers, or
-- You map the Docker socket into the container so that Renovate can dynamically invoke "sidecar" images when necessary
+If you want, you can map the Docker socket into the container so that Renovate can dynamically invoke "sidecar" images when needed.
+You'll need to set `binarySource=docker` for this to work.
+Read the [`binarySource` config option docs](../self-hosted-configuration.md/#binarysource) for more information.
 
-The "full" image (which `latest` defaults to) has every package manager which Renovate supports already preinstalled.
-This approach works best for many, but does have the following downsides:
+##### The default image
 
-- It only has _one_ version of each language/manager - usually the latest
-- It's several gigabytes in size
+The default image comes with most package managers that Renovate supports, but not _all_ package managers.
 
-The `renovate/renovate` Docker images are compatible with all of Renovate's supported platforms.
+You must set `binarySource=global`, if you don't then Renovate still installs the latest version of the tools or the requested version from the repository.
+The pre-installed tool are sometimes outdated if the user doesn't update the Docker image regularly.
+
+Renovate supports a persistent cache for downloaded tools, so that it only needs to unpack the tools on later runs.
+Use the [`containerbaseDir` config option](../self-hosted-configuration.md/#containerbasedir) to control where Renovate stores its containerbase cache.
```

Comment:
Do you want me to extract these two lines into a new heading, and put it somewhere suitable?

```markdown
### Persistent cache for downloaded tools

Renovate supports a persistent cache for downloaded tools, so that it only needs to unpack the tools on later runs.
Use the [`containerbaseDir` config option](../self-hosted-configuration.md/#containerbasedir) to control where Renovate stores its containerbase cache.
```

Reply:
I think this is maybe to much detail for a getting started. @viceice wdyt?

Reply:
yes, this is something advanced. Maybe we should have a new page about renovate caching. (redis, s3, containerbase, ...)

Reply:
When I search for `cache` or `caching` in the Renovate docs search I get hits over multiple pages.

Sounds good to have a single page to explain the whole "How Renovate uses caching" thing. I don't know anything about caching. One of the maintainers will have to write a draft that I can review. 😄

Reply:
@rarkins can you create an issue for us about creating a Renovate caching explanation page, and _what_ it should explain?

I don't know enough to create an actionable issue for you... 🙈

Reply:
https://github.com/renovatebot/renovate/issues/22284